dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java

```diff
@@ -116,7 +116,7 @@
     QUERY_TENANT_LIST_ERROR(10086, "query tenant list error", "查询租户列表错误"),
     UPDATE_TENANT_ERROR(10087, "update tenant error", "更新租户错误"),
     DELETE_TENANT_BY_ID_ERROR(10088, "delete tenant by id error", "删除租户错误"),
-    VERIFY_TENANT_CODE_ERROR(10089, "verify tenant code error", "租户编码验证错误"),
+    VERIFY_TENANT_CODE_ERROR(10089, "verify os tenant code error", "操作系统租户验证错误"),
```

Comment:
hi,suggest change

Reply:
TENANT_CODE is recommended to change to OS_TENANT_CODE

Reply:
got it